sdk/PrebidMobile/PBBidManager.m

```diff
@@ -194,13 +204,20 @@ - (void)attachTopBidHelperForAdUnitId:(nonnull NSString *)adUnitIdentifier
     }
 }
 
+-(void) loadOnSecureConnection:(BOOL) secureConnection {
+    if(self.adServer != PBPrimaryAdServerUnknown && self.adServer != PBPrimaryAdServerDFP){
```

Comment:
1. This was not implemented in the SDK before? how did SDK send secure request to prebid server?
2. I'd prefer if adServer == MoPub, allow override, since do not override is default.

Reply:
done